.github/workflows/manifest.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       run: |
         sudo apt-get -qqy install zstd
```

Comment:
We can delete these lines now, right? Because we end up installing zstandard and the requirements at runtime.

Reply:
I didn't try it, I assumed the command tool was needed to compress, but I can check if that's actually the case.

In two weeks :)

Reply:
We missed the flight, so I had time to do this now instead. I see that we use the `zstd` binary here:
https://github.com/web-platform-tests/wpt/blob/bc351be47b742e2d306907de797f6dd56093f092/tools/ci/manifest_build.py#L43

We could use https://python-zstandard.readthedocs.io/en/latest/misc_apis.html#compress, but while trying this out I discovered that these files aren't even smaller and have filed https://github.com/web-platform-tests/wpt/issues/33489.